static_frame/core/frame.py

```diff
@@ -187,13 +187,23 @@ def from_concat(cls,
         # axis_series = (0 if axis is 1 else 1)
         frames = [f if isinstance(f, Frame) else f.to_frame(axis) for f in frames]
 
+        own_columns = False
+        own_index = False
+
+        # End quickly if empty iterable
+        if len(frames) == 0:
```

Comment:
if not len(frames):, perhaps?

Reply:
How about `if not frames:` as frames is a Python list, and list are truthy.

Reply:
Nice!